crates/api_models/src/admin.rs

```diff
@@ -76,12 +76,7 @@ pub struct MerchantAccountCreate {
     pub locker_id: Option<String>,
 
     ///Default business details for connector routing
-    #[cfg(feature = "multiple_mca")]
     #[schema(value_type = PrimaryBusinessDetails)]
```

Comment:
This becomes a mandatory field now as per spec?!

Reply:
No, This will be an Optional field. I have updated the openapi file